packs/clicrud/actions/ops_command.py

```diff
@@ -0,0 +1,36 @@
+# encoding: utf-8
+from st2actions.runners.pythonrunner import Action
+from clicrud.device.generic import generic
+import sys
+
+
+class OPSCommand(Action):
+
+    def run(self, host, command):
+        self.method = self.config['method']
+        self.username = self.config['username']
+        self.password = self.config['password']
+        self.enable = self.config['enable']
+
+        self.method = self.method.encode('utf-8', 'ignore')
```

Comment:
I assume you call encode to make sure you we are operating with a string and not unicode? Or is there some other reason for it?